### ISSUE TITLE
roxygen2md converts Rd syntax within R code

### ISSUE THREAD

**Issue by karawoo** [r-lib/roxygen2md#4]
Not sure if this qualifies as a bug, but I noticed that `roxygen2md` converts Rd syntax that is present in R _code_ as well as documentation. A reprex:

``` r
setwd(tempdir())
devtools::create("test")
setwd("test")

cat(
  "#' Title",
  "#' ",
  "#' @param x,y Some stuff here, and a random \\code{code tag}",
  "test_fun <- function(x, y) {",
  "paste('\\code{\', x, \'} and \', y, sep = \'\')",
  "}",
  file = "R/test_fun.R",
  sep = "\n"
)

## View function
writeLines(readLines("R/test_fun.R"))
#> #' Title
#> #' 
#> #' @param x,y Some stuff here, and a random \code{code tag}
#> test_fun <- function(x, y) {
#> paste('\code{', x, '} and ', y, sep = '')
#> }

## Convert to md and view
roxygen2md::roxygen2md()
#> Please review the changes carefully!
writeLines(readLines("R/test_fun.R"))
#> #' Title
#> #' 
#> #' @param x,y Some stuff here, and a random `code tag`
#> test_fun <- function(x, y) {
#> paste('`', x, '` and ', y, sep = '')
#> }
```

I would expect the `\code{code tag}` in the documentation to get converted, but not contents of the function. Though in fairness I was warned to review the changes carefully!

This can come up when a function in the package writes Rd style output and gets called by a `\Sexpr` macro, like [these](https://github.com/tidyverse/ggplot2/blob/master/R/utilities-help.r) in ggplot2 (which is where I ran into this issue).

**Comment by krlmlr** [comment]
Thanks for reporting. It seems too late for ggplot2, but you could run again on `HEAD~2` and double-check just to be sure.

**Comment by krlmlr** [comment]
Closed in 5115d65d.
